File: script/make/protocol/json/CheatProtocol.js
const cheatProtocol = {
    read: {
        60001: [
            {name: "cheatList", type: "list", comment: "秘籍列表", explain: [
                {name: "description", type: "str", comment: "描述", explain: []},
                {name: "command", type: "str", comment: "命令", explain: []}
            ]}
        ],
        60002: [
            {name: "result", type: "rst", comment: "结果", explain: []}
        ]
    },
    write: {
        60001: [],
        60002: [
            {name: "command", type: "str", comment: "命令", explain: []}
        ]
    }
};

File: script/make/protocol/lua/meta/CheatProtocol.lua
return {
    [60001] = {
        ["comment"] = "秘籍",
        ["write"] = {name = "data", type = "map", comment = "", explain = {

        }},
        ["read"] = {name = "data", type = "list", comment = "命令列表", explain = {
            {name = "data", type = "map", comment = "", explain = {
                {name = "description", type = "bst", comment = "描述", explain = {}},
                {name = "command", type = "bst", comment = "命令", explain = {}}
            }}
        }}
    },
    [60002] = {
        ["comment"] = "秘籍",
        ["write"] = {name = "data", type = "bst", comment = "命令", explain = {}},
        ["read"] = {name = "data", type = "ast", comment = "结果", explain = {}}
    }
}